backend/tests/v5/unit/middleware/auth.test.js

```diff
@@ -34,22 +36,49 @@ const AuthMiddlewares = require(`${src}/middleware/auth`);
 // Mock respond function to just return the resCode
 Responder.respond.mockImplementation((req, res, errCode) => errCode);
 
+const token = generateRandomString();
+const session = { user: { referer: 'http://abc.com' } };
+const cookies = { [CSRF_COOKIE]: token };
+const headers = { referer: 'http://abc.com/', [CSRF_HEADER]: token };
+
 const testValidSession = () => {
```

Comment:
actually now that I see it , wouldnt it make more sense to mock isSessionValid and test the scenarios that it returns true or false ? Since we already have backend/tests/v5/unit/utils/sessions.test.js we are testing the same things twice.... I know you didnt add these but just an observation

Reply:
yea I also noticed this - we should really be testing that on utils/sessions and mocking the return values here. but in the interest of time I just followed the flow....

added as a maintenance task https://github.com/3drepo/3drepo.io/issues/5072

Reply:
> could add a test with CSRF_HEADER in capital case to check line 36 works

The constants is actually in upper case. I've provided a test case for both upper and lower so we're 100% covered.